backend/wmg/data/snapshot.py

```diff
@@ -73,8 +72,12 @@ class WmgSnapshot:
     filter_relationships: Optional[Dict] = field(default=None)
 
     # dataset metadata dictionary
+
     dataset_metadata: Optional[Dict] = field(default=None)
 
+    def __hash__(self):
```

Comment:
if the object doesn't have to hashable then why is this there? Could it be removed?